python/tests/core/test_feature_monitoring_result_engine.py

```diff
@@ -190,8 +203,10 @@ def test_save_feature_monitoring_result_via_fv(self, mocker):
         result = mock_result_api.call_args[0][0]
         assert result._config_id == DEFAULT_CONFIG_ID
         assert result._execution_id == execution_id
-        assert result._detection_stats_id == detection_stats_id
-        assert result._reference_stats_id == reference_stats_id
+        assert result._detection_stats_id is None
+        assert result._reference_stats_id is None
+        assert isinstance(result.detection_statistics, FeatureDescriptiveStatistics)
+        assert isinstance(result.reference_statistics, FeatureDescriptiveStatistics)
```

Comment:
I am not seeing a test where you provide only detection statistics, probably my fault since I was always passing both detection and reference id. We should make sure that unit test cover the scheduled statistics with detection only.

Reply:
I added a couple of tests with specific value as reference and scheduled statistics.